src/components/accessories/patientTherapy/PatientTherapy.tsx

```diff
@@ -55,7 +55,7 @@ const PatientTherapy: FC<TProps> = ({
 
   const onSubmit = (therapy: TherapyRowDTO) => {
     setShouldResetForm(false);
-    therapy.patID = undefined;
+    therapy.patID = patient;
```

Comment:
Maybe it would be better to name `patient` as `patientId`, in order to make it clearer.

Reply:
I replaced with patientId, but the confusion there comes from therapy api data model. Take a look: 
`
{
  "therapyID": 1,
  "patID": {
    "code": 0,
    .........
    }
    .........
}
`

Reply:
So the naming in api data model is not fine.